docs/how_to/use_sampler.rst

```diff
@@ -0,0 +1,248 @@
+#####################################################
+Compute circuit output probabilities with ``Sampler``
+#####################################################
+
+This guide shows how to get the probability distribution of a quantum circuit with the :class:`~qiskit.primitives.Sampler` primitive.
+
+.. note::
+
+    While this guide uses Qiskit’s reference implementation, the ``Sampler`` primitive can be run with any provider using :class:`~qiskit.primitives.BackendSampler`.
+    
+    .. code-block::
+
+        from qiskit.primitives import BackendSampler
+        from qiskit_provider import QiskitProvider
```

Comment:
ditto https://github.com/Qiskit/qiskit-terra/pull/9716#discussion_r1189680518

Reply:
Done in [3ee6158](https://github.com/Qiskit/qiskit-terra/pull/9716/commits/3ee61586908b0c6728b06d93d8c90d27c8431fe9)